Clinical-Genomics/cg
# Adds cli set microbial order

This PR changes the commands to use '-' instead of '_'

How to test:
0. install on stage (add instructions on how to as needed)
1. us
1. run cg set microbial-order
1. run cg set microbial-sample

Expected outcome:
both should be recognized as valid commands
